locales/es.po

```diff
@@ -17931,3 +17954,393 @@ msgstr "Inválido {0}: brote especificado en el índice {1}, pero sólo tenemos
+
+#: frontend/src/metabase/lib/expressions/helper_text_strings.js:467
+msgid "Checks if a column is null"
+msgstr "Comprueba si una columna es nula"
+
+#: frontend/src/metabase/lib/expressions/helper_text_strings.js:472
+#: frontend/src/metabase/lib/expressions/helper_text_strings.js:484
+msgid "The column to check."
+msgstr "La columna a comprobar."
+
+#: frontend/src/metabase/lib/expressions/helper_text_strings.js:479
+msgid "Checks if a column is empty"
+msgstr "Comprueba si una columna está vacía"
+
+#: frontend/src/metabase/lib/expressions/typechecker.js:43
+msgid "CASE expects 2 arguments or more"
+msgstr "CASE espera 2 argumentos o más"
+
+#: frontend/src/metabase/lib/expressions/typechecker.js:65
+msgid "Function {0} expects {1} argument"
+msgid_plural "Function {0} expects {1} arguments"
+msgstr[0] "La función {0} espera un argumento"
+msgstr[1] "La función {0} espera {1} argumentos"
+
+#: frontend/src/metabase/lib/expressions/typechecker.js:105
+#: frontend/src/metabase/lib/expressions/typechecker.js:114
+msgid "Expecting boolean but found {0}"
+msgstr "Se esperaba un valor booleano pero se encontró {0}"
+
+#: frontend/src/metabase/lib/time.js:83
+msgid "12:00 PM"
+msgstr "12:00 PM"
+
+#: frontend/src/metabase/meta/Dashboard.js:52
+msgid "Subtotal, Age, Price, Quantity, etc."
+msgstr "Subtotal, Edad, Precio, Cantidad, etc."
+
+#: frontend/src/metabase/meta/Parameter.js:73
+msgid "Dropdown"
+msgstr "Desplegable"
+
+#: frontend/src/metabase/meta/Parameter.js:74
+msgid "Select one or more values from a list or search box."
+msgstr "Selecciona uno o más valores de una lista o cuadro de búsqueda."
+
+#: frontend/src/metabase/meta/Parameter.js:79
+msgid "Exclude one or more specific values."
+msgstr "Excluye uno o más valores específicos."
+
+#: frontend/src/metabase/meta/Parameter.js:84
+msgid "Match values that contain the entered text."
+msgstr "Coincidir con los valores que contienen el texto introducido."
+
+#: frontend/src/metabase/meta/Parameter.js:89
+msgid "Filter out values that contain the entered text."
+msgstr "Filtra los valores que contienen el texto introducido."
+
+#: frontend/src/metabase/meta/Parameter.js:94
+msgid "Match values that begin with the entered text."
+msgstr "Coincidir con los valores que comienzan con el texto introducido."
+
+#: frontend/src/metabase/meta/Parameter.js:99
+msgid "Match values that end with the entered text."
+msgstr "Coincidir con los valores que terminan con el texto introducido."
+
+#: frontend/src/metabase/query_builder/components/view/QuestionEntityMenu.jsx:42
+msgid "Duplicate this question"
+msgstr "Duplica esta pregunta"
+
+#: frontend/src/metabase/search/components/SearchResult.jsx:170
+msgid "Saved question in {0}"
+msgstr "Pregunta guardada en {0}"
+
+#: frontend/src/metabase/search/components/SearchResult.jsx:176
+msgid "Table in {0}"
+msgstr "Tabla en {0}"
+
+#: frontend/src/metabase/search/components/SearchResult.jsx:212
+msgid "Segment of "
+msgstr "Segmento de "
```

Comment:
Hrm, https://github.com/metabase/metabase/blob/ce939219f8a3e28ecb00deb232154780a1dcfcef/frontend/src/metabase/search/components/SearchResult.jsx#L214-L221 isn't ideal, but too late now

(should be `Segment of {0}`, right?)

Reply:
It looks like it's doing `Segment of` (or `Metric for` and then the Table name separately. The Table name isn't wrapped in `<Link>` so you wouldn't want to splice that in to a string

Reply:
It's an awkward situation for the reason you said; I believe the correct Scottish Gaelic would be `foo na Seigmeant` instead of `Seigmeant na foo`, for example. (Not that we provide GD translations, but I'm sure the same logic applies to translations we do ship)

It's something we can live with for 39, but if there's an easy fix in the future it'd be nice. Isn't there a way to splice together rich text?